### DISCUSSION TITLE
Allow copy `account` In /authenticator note type

### DISCUSSION THREAD

**Discussion by eadoking** [standardnotes/forum#3301] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
yes I did 

**Is your feature request related to a problem? Please describe.**

not a problem by it self, but I believe it is a core important feature for usability.

Any time I am using the `authenticator` type to copy `password` or `2FA`, I am frustrated not to be able to copy the username/account.

I believe this is an important feature, since not all accounts are simple usernames.

they could be long, different and will make so much different to implement it.

**Describe the solution you'd like**
I created a sample note. type `authenticator` and added sample entries. 
- test-service-1
- test-service-2

for each, added 2FA + password

![CleanShot 2023-10-09 at 14 52 23](https://github.com/standardnotes/forum/assets/441839/ad0bac9b-fb3d-412a-b43c-a5d98b560f8a)

When I hover over the `password` I can click it, it is copied to the clipboard 
<img width="350" alt="image" src="https://github.com/standardnotes/forum/assets/441839/5e7c5e17-ad34-4528-bd3b-67d5fbe0ed29">

the same functionality works when clicking the 2FA code.

**REQUESTED**
copying to clipboard should be also when clicking on the account/username for both 
- 2FA entry type
- password entry type

**Describe alternatives you've considered**
no alternatives 

**Additional context**

**Comment by shompoe** [comment]
I too find this fustrating, and in addition would like to suggest a separate URL field. Then when the service name is tapped, the browser could be launched straight to the login page of that service.
